pyproject.toml

```diff
@@ -9,6 +9,7 @@ readme = "README.md"
 python = "^3.10"
 boto3 = "^1.33.10"
 fastapi = "^0.104.1"
+ruff = "^0.1.8"
```

Comment:
ruff is not the project dependency, it is development environment dependency.

To configure it like that let's use dependency groups (dev dependencies)

https://python-poetry.org/docs/managing-dependencies/#dependency-groups